src/hooks/useTodayDate.ts

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { getMillisecondsToTomorrow } from "../lib/date";
+
+/**
+ * Обновляемая дата сегодняшнего дня
+ *
+ * Дата - сегодня (в соответствии с системным временем)
+ *
+ * Часы, минуты, секунды, миллисекунды - произвольные
+ */
+export function useTodayDate() {
```

Comment:
```suggestion
export function useTodayDate(listenDayChangesForUpdate: boolean = false) {
```

Блин, сорри, из головы вылетело...

Давай добавим аргумент, который вкл/выкл автоматическое обновление. Не всем может быть это надо.

Надо такой же завести у `Calendar`

```ts
interface CalendarProps {
  listenDayChangesForUpdate?: boolean;
}
```

По умолчанию тоже `false`